src/utils/events/createLoadSafe.ts

```diff
@@ -0,0 +1,70 @@
+import { GTM_EVENT, trackEvent } from 'src/utils/googleTagManager'
+
+// Legacy GA events encapsulated the entire stepper under 'Created a Safe'
```

Comment:
Legacy means we're going to deprecate them, is that the plan?

Reply:
After confirming on Slack: all will be added [here](https://github.com/gnosis/safe-react/pull/3675).